src/Microsoft.Health.Dicom.Functions/Update/UpdateDurableFunction.Orchestration.cs

```diff
@@ -44,16 +45,23 @@ public partial class UpdateDurableFunction
 
         UpdateCheckpoint input = context.GetInput<UpdateCheckpoint>();
 
+        UpdateOperationInput updateInput = await context.CallActivityWithRetryAsync<UpdateOperationInput>(
```

Comment:
Inserting an activity is a breaking change, and if we're worried about any "in-flight" updates during deployment, then we need to "version" it by updating the orchestration name

Reply:
Yeah thats true, since no one is using. I am not worried about versioning now